integration-tests/integration/activity/selectActivityLocation.spec.js

```diff
@@ -1,23 +1,73 @@
+const moment = require('moment')
+
+const caseload = 'MDI'
+const date = new Date().toISOString().split('T')[0]
+
+// copy of { getCurrentPeriod } from backend/utils/.js
+const getCurrentPeriod = now => {
```

Comment:
Temporary for now. I'll raise a tech debt ticket so that we can make the Wiremocks stubs more lenient.